glean-core/android/metrics.yaml

```diff
@@ -208,39 +208,52 @@ glean.internal.metrics:
     expires: never
 
 glean.error:
- invalid_value:
-   type: labeled_counter
-   description:
-     Counts the number of times a metric was set to an invalid value.
-     The labels are the `category.name` identifier of the metric.
-   bugs:
-     - https://bugzilla.mozilla.org/1499761
-   data_reviews:
-     - https://bugzilla.mozilla.org/show_bug.cgi?id=1499761#c5
-   notification_emails:
-     - telemetry-client-dev@mozilla.com
-   expires: never
-   send_in_pings:
-     - all_pings
-   no_lint:
-     - COMMON_PREFIX
+  invalid_value:
+    type: labeled_counter
+    description:
+      Counts the number of times a metric was set to an invalid value.
+      The labels are the `category.name` identifier of the metric.
+    bugs:
+      - https://bugzilla.mozilla.org/1499761
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1499761#c5
+    notification_emails:
+      - telemetry-client-dev@mozilla.com
+    expires: never
+    send_in_pings:
+      - all_pings
+    no_lint:
+      - COMMON_PREFIX
```

Comment:
This just fixes the indentation.

Reply:
huh, wonder why these were all wrong to begin with.

Reply:
/me goes to add yamllint (if that even exists) to CI...